packages/sdk-zodiac-nomad-module/src/abis/NomadModule.json

```diff
@@ -0,0 +1,468 @@
+{
```

Comment:
can we build this with typechain and commit the output?

Reply:
Gonna delete this actually. The `exec` function is private, so it's not included in the abi. At least, I'm assuming that's the reason